x-pack/plugins/security/server/lib/mirror_status_and_initialize.js

```diff
@@ -25,36 +25,28 @@ export function mirrorStatusAndInitialize(upstreamStatus, downstreamStatus, onGr
 
```

Comment:
We can delete this file and the associated tests, since we aren't using it anymore.

Reply:
I left this in because I believe Spaces will need it to create the default space on startup. I can just as easily port it over to phase 2 if you'd rather see it removed from phase 1

Reply:
Ah, nope, feel free to leave it then, but you probably want to revert it to how it used to be.